bin/git-utimes

```diff
@@ -55,23 +48,30 @@ if git status --help 2>&1 | grep -q -- "--ignored"; then
   status_opts="${status_opts} --ignored=no"
 fi
 
+prefix="$(git rev-parse --show-prefix) "
+strip="${#prefix}"
+
 tmpfile=$(mktemp)
+# shellcheck disable=SC2064
 trap "rm -f '${tmpfile}'" 0
 
 # prefix is stripped:
+# shellcheck disable=SC2086
 git --no-pager status --porcelain --short ${status_opts} . |
   cut -c 4- >"${tmpfile}"
 
 # prefix is not stripped:
+# shellcheck disable=SC1003,SC2086,SC2248
 git --no-pager whatchanged ${whatchanged_opts} --format='%ct' . |
-  awk $awk_flags \
+  awk ${awk_flags} \
     -F'\t' \
     -v date_flags="${date_flags}" \
     -v op="${op}" \
     -v stat_flags="${stat_flags}" \
     -v strip="${strip}" \
     -v tmpfile="${tmpfile}" \
-    'BEGIN {
+    '\
```

Comment:
I don't think this backslash is needed?

Reply:
@hyperupcall It isn’t. I added it as `shfmt` was indenting in a way my editor flagged as an error. We can remove if you want.

Reply:
Ah I see it's a workaround then - no worries then

Reply:
@rasa 
Could you add the reason as comment so other people can understand it?

Reply:
@spacewander I removed the weird formatting instead.